sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -12,6 +12,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableClientSdkAnalyzers>true</EnableClientSdkAnalyzers>
     <EnableBannedApiAnalyzers>false</EnableBannedApiAnalyzers>
+
```

Comment:
It would be helpful to revert this file if there are no changes to it other than whitespace.